lib/utils/utils.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+class Strings {
```

Comment:
Rename to UtilsStrings or AppStrings or something more meaningful, same for Text_Styles.
For class name use AppColors instead of App_Colors, same for other classes

Reply:
Ok